mysql-test/main/subselect_nulls_innodb.result

```diff
@@ -0,0 +1,26 @@
+#
+# MDEV-32090 Index does not handle null-safe equals operator correctly in join
```

Comment:
I would also add "(testcase only)" a the end of this line as well as at the end of the first line of the commit comment.